pkg/pipeline/PipelineBuilder.go

```diff
@@ -1413,6 +1418,24 @@ func (impl PipelineBuilderImpl) PatchCiPipeline(request *bean.CiPatchRequest) (c
 	ciConfig.AppWorkflowId = request.AppWorkflowId
 	ciConfig.UserId = request.UserId
 	if request.CiPipeline != nil {
+		client, err := impl.K8sUtil.GetClientForInCluster()
```

Comment:
will it works only for in cluster ? if not handle by cluster id.

Reply:
Yes working .